DatadogInternal/Sources/DatadogFeature.swift

```diff
@@ -37,7 +37,7 @@ public protocol DatadogFeature {
     ///
     /// The `FeatureMessageReceiver` defines an interface for Feature to receive any message
     /// from a bus that is shared between Features registered in a core.
-    var messageReceiver: FeatureMessageReceiver { get }
+    var messageReceiver: FeatureMessageReceiver? { get }
```

Comment:
Similarly to performanceOverride, not all feature are initially interested in using messageReceiver.